feature/connections/README.md

```
@@ -120,18 +120,19 @@ Route: `ConnectionsRoute.Connections`
 
 ## Dependency Graph
 
+### Key Dependencies
+
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)